packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -448,26 +453,36 @@ export class Database {
       )
     }
 
-    const {
-      edges,
-      pageInfo: { hasPreviousPage, hasNextPage, startCursor, endCursor },
-    }: { edges: { path: string; cursor: string }[]; pageInfo: PageInfo } =
-      await this.store.query(storeQueryOptions)
-
-    return {
-      edges: await sequential(edges, async (edge) => {
-        const node = await hydrator(edge.path)
-        return {
-          node,
-          cursor: btoa(edge.cursor),
-        }
-      }),
-      pageInfo: {
-        hasPreviousPage,
-        hasNextPage,
-        startCursor: btoa(startCursor),
-        endCursor: btoa(endCursor),
-      },
+    try {
+      const {
+        edges,
+        pageInfo: { hasPreviousPage, hasNextPage, startCursor, endCursor },
+      }: { edges: { path: string; cursor: string }[]; pageInfo: PageInfo } =
+        await this.store.query(storeQueryOptions)
+
+      return {
+        edges: await sequential(edges, async (edge) => {
+          try {
+            const node = await hydrator(edge.path)
+            return {
+              node,
+              cursor: btoa(edge.cursor),
+            }
+          } catch (error) {
+            console.error(`error in fetching ${edge.path}`)
```

Comment:
Rather than logging this out here, would it be better to just throw a new Error with the more contextual message? This way you don't get the bubbling up of console.log statements:

```
error in fetching content/posts/anotherPost.mdx
error in querying posts
```
And instead could have one error that actually makes sense like
```
AuditError: Error while reading "content/posts/anotherPost.mdx"
  Could not parse expression with acorn: Unexpected token
    ... rest of stack trace
```
Also it might be nice to only show the stack trace when a `--verbose` flag is passed

Reply:
@jeffsee55 Agreed, fixed.